.github/workflows/handle_release_tag.yaml

```diff
@@ -0,0 +1,26 @@
+name: Handle Release Tag
+
+on:
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+  create_release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-tags: true
+        fetch-depth: 0
+    - name: Repair tag
```

Comment:
Seems like this is "Verify tag" vs "Repair tag"

Reply:
The actions/checkout manages to lose the "annotated" part of annotated tags.  This fetch will fix it by pulling the annotated part.